core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +29,8 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  public static Set<String> blobTypes() {
```

Comment:
is it supposed to return "all standard blob types"?
should the name reflect that?

if we did https://github.com/apache/iceberg/pull/8202, would this new blob type be added to this method?